apps/beeai-ui/src/modules/runs/chat/Chat.module.scss

```diff
@@ -26,7 +30,22 @@
 }
 
 .header {
-  padding-block: $spacing-03;
+  .isNew & {
+    padding-block: $spacing-03;
+    display: flex;
+    flex-direction: column;
+    gap: $spacing-06;
+    h1 {
+      margin: 0;
+    }
+  }
+}
+
+.heading {
+  font-size: rem(42px);
+  line-height: math.div(60, 42);
+  font-weight: 300;
+  @include font-family(serif);
```

Comment:
Let's put include first:
```scss
.heading {
  @include font-family(serif);
  // ...
}
```

**Reasons:** Easier to scan, Common convention, Reduces override risk.